definitions/variable/energy.yaml

```
@@ -18,6 +18,9 @@
     description: Consumption of {Electricity Input} for electricity generation
       (including transmission losses)
     unit: TJ
+- Conversion Input|{Alternative Fuel Output}|{Alternative Fuel Input}:
+    description: Consumption of {Synfuel Input} for generation of {Synfuel Output}
```

I think you mean the following?
```suggestion
- Conversion Input|{Alternative Fuel Output}|{Synfuel Input}:
    description: Consumption of {Synfuel Input} for generation of {Alternative Fuel Output}
```

---

```suggestion
- Conversion Input|{Alternative Fuel Output}|{Alternative Fuel Input}:
    description: Consumption of {Alternative Input} for generation of {Alternative Output}

```